docs/sources/tempo/operations/manage-advanced-systems/generic_forwarding.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Generic forwarding
+menuTitle: Generic forwarding
+description: Asynchronous replication of ingested traces
+weight: 400
+aliases:
+  - ../generic_forwarding/ # https://grafana.com/docs/tempo/<TEMPO_VERSION>/operations/consistent_hash_ring/
+---
+
+# Generic forwarding
+
+Generic forwarding allows asynchronous replication of ingested traces.
+The distributor writes received spans to both the ingester and defined endpoints, if enabled.
+This feature works in a best-effort manner, meaning that no retries happen if an error occurs during replication.
+
+{{< admonition type="warning" >}}
+Generic forwarding doesn't work retroactively. After it's enabled, the distributor only replicates freshly ingested spans.
+{{< /admonition >}}
+
+## Configure generic forwarding
+
+Enabling generic forwarding requires the configuration of the `distributor` and `overrides`.
+
+1. First, define a list of forwarders in the `distributor` section. Each forwarder must specify a unique `name`, supported `backend`, and backend-specific configuration.
```

Comment:
not sure you need "first" and "second" because the steps are numbered.

Reply:
```suggestion
1. Define a list of forwarders in the `distributor` section. Each forwarder must specify a unique `name`, supported `backend`, and backend-specific configuration.
```